src/main/webapp/assets/script.js

```diff
@@ -11,6 +11,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+/* eslint-disable no-unused-vars */
```

Comment:
L I N T E R B E G O N E

Reply:
b a n i s h ' d
(it'll be removed as soon as we have the surrounding infra to call `translateLocationToLatLong` appropriately! 😀 )

Reply:
LMAO!!